From: Jan Grulich <jgrulich@redhat.com>
Date: Thu, 17 Sep 2015 14:14:24 +0000
Subject: Use Q_SLOTS/Q_SIGNALS instead of slots/signals in all headers from include dir
X-Git-Url: http://quickgit.kde.org/?p=qca.git&a=commitdiff&h=66b9754170759d9333d5fc1e348642814d0310dd
---
Use Q_SLOTS/Q_SIGNALS instead of slots/signals in all headers from include dir
REVIEW:125289
---


--- a/include/QtCrypto/qca_safetimer.h
+++ b/include/QtCrypto/qca_safetimer.h
@@ -44,12 +44,12 @@
 	void setSingleShot(bool singleShot);
 	int timerId() const;
 
-public slots:
+public Q_SLOTS:
 	void start(int msec);
 	void start();
 	void stop();
 
-signals:
+Q_SIGNALS:
 	void timeout();
 
 protected:

